package/firmware/ath10k-ct-firmware/Makefile

```diff
@@ -1,9 +1,11 @@
-include $(TOPDIR)/rules.mk
-
 PKG_NAME:=ath10k-ct-firmware
 PKG_VERSION:=2020-11-08
 PKG_RELEASE:=1
 
+DL_MKDIR:=1
```

Comment:
Should we document this in the wiki?

Reply:
I suppose, but I hope the commit message is self-explanatory enough

Reply:
I personally first check the makefile wiki page to search if a special magic value can do what I need so yes adding this to the wiki is mandatory :D